EC_process/timeline.md

```diff
@@ -0,0 +1,54 @@
+# Executive Council Yearly Timeline
+
+Additional tasks that don't fit into the monthly timeline below include:
+* in-person (face-to-face) meeting
+* reassessing the strategic plan
+* annual finance audit
+
+## January
```

Comment:
We maybe want to collect availability for the in-person meeting if we want to have it on the begin of the year. Otherwise, every member calendar will be full.

Reply:
Added both of these to January!